Modules/FT0/etc/recpoints_monitoring.json

```diff
@@ -0,0 +1,64 @@
+{
+    "qc": {
+      "config": {
+        "database": {
+          "implementation": "CCDB",
+          "host": "http://j183.localdomain:8083",
+          "username": "not_applicable",
+          "password": "not_applicable",
+          "name": "not_applicable"
+        },
+        "Activity": {
+          "number": "42",
+          "type": "2"
+        },
+        "monitoring": {
+          "url": "infologger:///debug?METRIC"
+        },
+        "consul": {
+          "url": "http://j183.localdomain:8500"
+        },
+        "conditionDB": {
+          "url": "http://j183.localdomain:8083"
+        }
+      },
+      "tasks": {
+        "RecPointsQcTask": {
+          "active": "true",
+          "className": "o2::quality_control_modules::ft0::RecpointsQcTask",
+          "moduleName": "QcFT0",
+          "detectorName": "FT0",
+          "cycleDurationSeconds": "10",
+          "maxNumberCycles": "-1",
+          "dataSource": {
+              "type": "direct",
+              "query": "recpoints:FT0/RECPOINTS/0;channels:FT0/RECCHDATA/0"
+             },
+          "taskParameters": {
+            "ChannelIDs": "73,76,79,83,122,125,129,132"
+          },
+          "location": "local",
+          "localMachines": [
+            "j183.localdomain"
+          ],
+          "remoteMachine": "j183.localdomain",
+          "remotePort": "30132",
+          "mergingMode": "entire"
```

Comment:
The parameters from line 40 to 46 are needed for multinode setups (e.g. many QC Task instances and merged objects). Just to make sure, do you need it?